New Unreal features support and known issues pages (#868)
spatialos/UnrealGDK: SpatialGDK/Documentation/known-issues.md

<%(TOC)%>
# Known issues

For the current state of support of Unreal features, including any workarounds for features which are not fully supported yet, see the [Unreal features support page]({{urlRoot}}/unreal-features-support).

> For all other Known Issues, please visit [this board](https://github.com/spatialos/UnrealGDK/projects/2) on Github. Feel free to contribute and subscribe to any issue, and raise new ones from [this page](https://github.com/spatialos/UnrealGDK/issues).

Reviewer: ```suggestion
For all other known issues, please visit [this board](https://github.com/spatialos/UnrealGDK/projects/2) on Github. Feel free to contribute and subscribe to any issue, and raise new ones from [this page](https://github.com/spatialos/UnrealGDK/issues).
```

Reviewer: We avoid proper nouns where we can because it's a mare to maintain (see Unreal or Unity docs!).

<%(TOC)%>

> # Known issues

Reviewer: As we are changing the contents of this page, we need to add the update info at the bottom.
```
<br/>

------------
2019-04-25 Page updated with full editorial review </br>
2019-04-25 Removed out-dated known issues from this page and added links to current known issues.
<br/>

```

For the current state of support of Unreal features, including any workarounds for features which are not fully supported yet, please visit the [Unreal features support page]({{urlRoot}}/unreal-features-support).

> For all other Known Issues, please visit [this board](https://github.com/spatialos/UnrealGDK/projects/2) on Github. Feel free to contribute and subscribe to any issue, and raise new ones from [this page](https://github.com/spatialos/UnrealGDK/issues).

Reviewer: ```suggestion
For all other known issues, see [this board](https://github.com/spatialos/UnrealGDK/projects/2) on Github. You can contribute and subscribe to any issue, and raise new ones from [the repository's issues page](https://github.com/spatialos/UnrealGDK/issues).
```
It's too nice "feel free". Docs are instructional and bossy.

Author: fair play !
